doc/source/tune/examples/pbt_transformers.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+pbt_transformers_example
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. literalinclude:: /../../python/ray/tune/examples/pbt_transformers/pbt_transformers.py
```

Comment:
Can you also literalinclude all of pbt_transformers/ files?

Reply:
done.